### PR TITLE
Print out basic pipeline skeleton

### PR DESCRIPTION
**Pull request by hliatis**
Prints out a basic skeleton in GAPIT for dump_pipelines

### CODE REVIEW

**Comment by hliatis** [comment]
Bug: 141620836

File: cmd/gapit/dump_pipeline.go [hunk 1]
```diff
@@ -79,9 +77,9 @@ func (verb *pipeVerb) getBoundPipelineResource(ctx context.Context, c client.Cli
 		return nil, err
 	}
 
-	targetType := api.Pipeline_GRAPHICS
+	targetType := "Graphics Pipeline"
```
Comment (pau-baiget) [inline cmd/gapit/dump_pipeline.go]: Why are we moving back to string literals?

Comment (hliatis) [reply cmd/gapit/dump_pipeline.go]: Andrew believes it will make it more extensible.

Comment (pmuetschard) [reply cmd/gapit/dump_pipeline.go]: I don't think this is going to need extending anytime soon. I can understand strings for the stages, but the pipeline type should stay an enum.

**Pull request closed without merging**